src/pages/index.js

```diff
@@ -69,7 +69,8 @@ export default () => {
           </div>
           <div className={styles.list}>
             <h1>
-              {secondCity === "ROCK 5B" ? 'ROCK 5 Model B' : secondCity === "ROCK 5A" ? 'ROCK 5 Model A' : secondCity === "ROCK 3C" ? 'ROCK 3 Model C' : null}
+                          {secondCity === "ROCK 5B" ? 'ROCK 5 Model B' : secondCity === "ROCK 5A" ? 'ROCK 5 Model A' : secondCity === "ROCK 3C" ? 'ROCK 3 Model C' : secondCity === "ROCK 3A" ? 'ROCK 3 Model A' :
+                              secondCity === "ROCK 4SE" ? 'ROCK 4SE' : secondCity === "ROCK 4C+" ? 'ROCK 4C+' : secondCity === "ROCK 4C Plus" ? 'ROCK 4C Plus' : null}
```

Comment:
```suggestion
             {productFullName[secondCity]}
```
然后在`export default () => {`下面添加一个Dictionary定义：
```javascript
const productFullName = {
  "ROCK 5B": "ROCK 5 Model B",
  "ROCK 5A": "ROCK 5 Model A",
 ...
};
```